source/ShopPanel.h

```diff
@@ -210,10 +210,12 @@ class ShopPanel : public Panel {
 	void MainDown();
 	void CategoryAdvance(const std::string &category);
 	std::vector<Zone>::const_iterator Selected() const;
-	// Check if the given point is within the button zone, and if so return the
-	// letter of the button (or ' ' if it's not on a button).
 	char CheckButton(int x, int y);
 
+	bool EscortSelected();
+	bool CanPark();
+	bool CanUnpark();
```

Comment:
This could use some comments.

```suggestion
	// Check if at least one of the selected ships is an escort.
	bool EscortSelected();
	// Check whether there is a selected ship that can be parked/unparked.
	bool CanPark();
	bool CanUnpark();
```